### DISCUSSION TITLE
Couverture législative ⚖

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2609] [General]
Ce ticket contient une liste générale des dispositifs existants dans la législation française en matière de droit de la sécurité sociale, droit fiscal, et plus partiellement en droit du travail. Cette liste n'est évidemment pas exhaustive et privilégie les dispositifs connexes (au sens très large) au contenu de Mon-entreprise.fr. Les dispositifs listés n'ont toutefois pas tous vocation à être intégrés sur Mon-entreprise.fr, et cette liste ne constitue donc pas une “roadmap”. Elle permet néanmoins d'avoir un aperçu synthétique des sujets couverts et de ceux non couverts dans notre base de règles en l'état actuel des choses.

> **Note** : Il s'agit d'une liste évolutive, des éléments pourront s'ajouter, des catégories se réorganiser, et la progression être rapportée plus finement (actuellement elle ne l'est qu'avec une simple case à cocher). On pourra aussi référencer les tickets traitant de l'implémentation de dispositifs spécifiques.

## Formes de travail 📃
- [x] Salariés et assimilés
    - [x] CDI Contrat à Durée Indéterminée
    - [x] CDD Contrat à Durée Déterminée
    - [ ] CDIC CDI de chantier
    - [ ] CUI Contrat unique d'insertion
    - [ ] CTT Contrat de Travail Temporaire (intérim)
    - [x] Contrat d’apprentissage
    - [x] Contrat de professionnalisation
    - [x] Convention de stage
    - [ ] Doctorat
    - [ ] Travailleurs à domicile
    - [ ] Assistants maternels et familiaux
- [x] Indépendants
    - [x] Auto-entrepreneurs
    - [x] Artisan, commerçant
    - [x] Professions libérales
    - [x] Professions libérales réglementés (medecin, avocat, notaire, etc.) 
    - [x] Dirigeants assimilés salarié
    - [ ] Agriculteurs
    - [x] Artistes-auteurs #720
    - [ ] Marins 
    - [x] Conjoint collaborateur #727 
- [ ] Fonction publique
    - [ ] Titulaires
    - [ ] Non titulaires (contractuels, auxiliaires, vacataires, etc.)
    - [ ] Élus
    - [ ] Militaires

## Formes sociales 🏢
- [x] Micro-entreprise
- [x] Entreprise individuelle (EI, EIRL)
- [x] Sociétés anonymes avec un gérant indépendant (SARL, EURL)
- [x] Sociétés anonymes avec un président assimilé salarié (SA, SAS, SASU)
- [ ] Entreprises d'exercice libéral (SELARL, SELAFA, SELCA, SELAS, SELASU)
- [ ] Sociétés civiles (SCI, SCP, SCCV, SEP) 
- [ ] Sociétés de personnes (SNC, SCS)
- [ ] Coopératives (SCOP, SCIC, SCA) #181
- [ ] Exploitations agricoles (EARL, GAEC, SCEA)
- [ ] Societas europaea (SE)
- [ ] Associations #585 (loi 1901, AARPI)
- [ ] Sociétés d'économie mixte (SAEM, SAEML, SAIEM)
- [ ] Sociétés d'investissement et titrisation (SIIC, FCT, ST)
- [ ] Groupements (GIE, GIP, GFF, GFI)
- [ ] Établissements publics (EPA, EPIC)
- [ ] Fondations, fonds de dotation

## Cotisations sociales ☂
- [x] Maladie
    - [x] Complémentaire sante
    - [x] Accidents du Travail et Maladies Professionnelles
- [x] Vieillesse, invalidité/décès
    - [x] AGIRC-ARCO
    - [x] contribution d'équilibre général
    - [x] contribution d'équilibre technique
    - [x] Forfait social
    - [x] Auto-entrepreneurs
    - [x] Indépendants (SSI)
    - [ ] Caisses des professions libérales
        - [x] CIPAV
        - [x] CNBF (avocats) #847
        - [ ] CAVOM (officiers ministeriels)
        - [ ] CPRN (notaires)
        - [x] CAVEC (experts-comptables)  #847
        - [ ] CAVAMAC (agents d'assurances)
        - [x] CARMF (médecins) #1103
        - [x] CARCDSF (dentistes, sages-femmes) #1103
        - [x] CAVP (pharmaciens) #1679
        - [x] CARPIMKO (infirmiers, kinésithérapeutes) #1103
        - [ ] CARPV (vétérinaires)  #1753
    - [x] IRCEC (complémentaire artistes-auteurs) #1478
    - [x] Contrats Madelins #533
- [x] Allocations familiales
- [x] Assurance chômage
    - [x] APEC
- [x] Logement
    - [x] Fonds national d’aide au logement (FNAL)
    - [x] Effort de construction (1% logement)
- [x] Régime de Garantie des Salaires (AGS)
- [x] Versement transport
- [x] Formation
    - [x] Taxe d'apprentissage
    - [x] Contribution supplémentaire à l'apprentissage (CSA)
    - [x] Contribution formation spécifique (pour les CDD)
- [x] Taxe sur les salaires
- [x] Contribution au dialogue social
- [x] Déduction forfaitaire spécifique #679 
- [x] Taux réduits journalistes
- [x] CSG/CRDS
    - [x] Abattement assiette CSG/CRDS
- [x] Prélèvement à la source
    - [x] Taux neutre
    - [x] Taux neutre outre-mer
    - [x] Taux personalisé
    - [x] Abattement contrats court

## Aides 🦿
- [ ] Création-reprise d'entreprise
    - [x] Jeune entreprise innovante (JEI)
    - [ ] Jeune entreprise universitaire (JEU)
    - [ ] Crédit d'impôt Recherche (CIR)
    - [x] Aide à la création ou à la reprise d'une entreprise (Acre)
    - [ ] Aide à la reprise ou à la création d'entreprise (Arce)
    - [ ] Aide aux librairies indépendantes
    - [ ] Zones de revitalisation rurale (ZRR)
    - [x] Zones franches urbaines (ZFU)
    - [ ] Zones de restructuration de la défense (ZRD)
    - [ ] Exonération CFE prof. lib. des communes de moins de 2000 habitants
    - [ ] Cumul avec l’allocation d’aide au retour à l’emploi (ARE)
- [ ] Embauche-formation
    - [x] Réduction générale (ex réduction Fillon)
    - [x] Réductions sur les heures supplémentaires
    - [x] Aide à l'embauche d'apprentis
    - [x] Franchise de cotisation stages et apprentissage
    - [ ] Parcours Emploi Compétence (PEC) #162
    - [x] Exonération Lodéom (outre-mer) #218
    - [x] Emplois francs #742
    - [ ] Emplois Réunion
    - [ ] Contrat unique d’insertion (CUI)
    - [ ] Crédit d'impôt pour la formation des dirigeants
    - [ ] Prime d'activité
    - [x] Allocation de chômage partiel
    - [ ] Aide embauche jeune artiste
    - [ ] Aides à l'embauche de personnes en situation de handicap (agefiph)
- [x] Prise en charge par l'État du chômage partiel #926 

## Rémunérations 💶
- [ ] Salaire minimum
    - [x] SMIC
    - [ ] SMIC jeune
    - [x] Gratification minimale
    - [ ] Rémunération de l'apprenti
- [ ] Avantages en nature, frais professionnels
    - [x] Nourriture, cantine
    - [x] Tickets restaurants, frais professionnels #243 
    - [ ] Voyage professionnel
    - [x] Smartphone, ordinateur (NTIC)
    - [x] Complémentaire santé
    - [ ] Logement #692 
    - [ ] Véhicule de fonction
    - [x] Prime vélo
    - [ ] Barème kilométrique
- [ ] Épargne salariale (PEE, PEI, PERCO)
- [ ] Indemnités
    - [x] fin de contrat
    - [x] indemnité kilométrique vélo
    - [x] compensation pour congés non pris
    - [x] indemnité journalières maladie
    - [x] chômage partiel #926 
    - [ ] non-concurrence
    - [ ] départ volontaire
- [ ] Primes
    - [ ] Prime d'ancienneté
    - [x] Prime de précarité
    - [x] Primes exceptionnelles #495
    - [x] Prime de treizième mois
    - [x] Régime des impatriés
    - [ ] Régime des expatriés
- [ ] Intéressement, participation
    - [ ] Dividendes
    - [ ] Participation aux bénéfices
    - [ ] Actions gratuites
    - [ ] Stocks options

## Conditions de travail ⏲
- [ ] Temps de travail
    - [ ] Forfait jour #1213
    - [x] Temps partiel
    - [x] Heures complémentaires #869 
    - [x] Heures supplémentaires
    - [ ] Journée de solidarité
    - [ ] Travail du dimanche
    - [ ] Travail de nuit
- [ ] Repos
    - [ ] Congés payés
    - [ ] Jours fériés
    - [ ] Arrêt maladie
    - [ ] Maternité, Congé parental
    - [ ] Événements (mariage, décès d'un proche, etc.)
- [x] Médecine du travail
- [ ] Périodes
    - [ ] Période d'essai
    - [ ] Préavis
    - [ ] Grève
- [ ] Télétravail

## Conventions collectives 📚
- [x] Hôtel café restaurant (HCR)
- [x] Optique #749 
- [x] Sportifs #757
- [x] Intermittents du spectacle #757
- [x] BTP #920

## Fiscalité société 🏛
- [x] Impôt sur les sociétés (IS)
- [x] Option micro-fiscal (micro-BNC, micro-BIC)
- [ ] SAS(U) à l'IR #1779 
- [ ] Contribution sociale sur l'IS
- [ ] Taxes pour frais de chambre
- [ ] TVA
- [ ] CVAE
- [ ] C3S
- [ ] CFE #751 

## Fiscalité particulier 👨‍👩‍👧‍👦
- [x] Barème de l'impôt sur le revenu
- [x] Décote de l'IR
- [x] Quotient familial
- [x] Frais professionnels #243 
- [x] Prélèvement libératoire de l'IR pour les auto-entrepreneurs
- [x] Contribution sur les hauts revenus (CEHR)
- [ ] Prélèvement forfaitaire unique (PFU)
- [ ] Conjoint collaborateur / SARL de famille 
- [x] Abattement journaliste #911

## Évolutions législatives 📆
- [x] Au 1<sup>er</sup> janvier 2019 #420
- [x] Au 1<sup>er</sup> octobre 2019 #435 
- [x] Au 1<sup>er</sup> janvier 2020 #676

> **Note** : la réflexion sur l'organisation des règles pourra servir pour la hiérarchie des espaces de noms dans la base de règles, ainsi que pour générer automatiquement une page d'accueil de la documentation avec des sections thématiques (cf. https://www.service-public.fr). Idéalement, la liste ci-dessus devrait être générée automatiquement à partir de la base de règles.

**Comment by laem** [comment]
Impressionnant ! Tu avais vu ce fichier qui traîne dans le dépôt depuis la nuit des temps 😄 ? Je crois que tu viens de lui donner un digne successeur.

https://github.com/betagouv/mon-entreprise/blob/master/source/r%C3%A8gles/brouillons/r%C3%A9gimes-juridiques.yaml

**Comment by mquandalle** [comment]
Ah oui effectivement, j'étais déjà tombé dessus mais je l'avais oublié. C'est plus complet et mieux organisé que ma liste « contrats de travail » donc à reprendre ici !
**Edit**: c'est fait ✅

**Comment by laem** [comment]
Je pense que la liste devrait être présente sur le site quelque part. Au moins dans la futur section "intégration", en en-tête de la /documentation, et même sur une page /à-propos qui dit qui est mon-entreprise et où on va. #709 @johangirod peut-être à intégrer sur le nouveau design ?

**Comment by mquandalle** [comment]
Oui idéalement je voudrais faire de cette liste la page d'accueil de la section `/documentation` (avec des blocs "sections" plutôt qu'une liste de listes), et la générer autant que possible à partir de la base de règles. En attendant l'avoir dans un format document texte permet d'itérer plus facilement (contenu, organisation des sections).

**Comment by laem** [comment]
@guillett ça peut t'intéresser

**Comment by mquandalle** [comment]
Cette liste n'est pas systématiquement maintenue à jour, et on a bien avancé sur la couverture législative depuis 2 ans. Je ferme mais reste le besoin de créer une page d’accueil de la doc #1082.

**Comment by mquandalle** [comment]
En fait je ré-ouvre parce qu'en attendant d'avoir ce contenu directement sur le site, ça reste un bon aperçu des sujets que l'on traite.
